src/controller/c/robot.c

```diff
@@ -1196,8 +1196,11 @@ static char *compute_socket_filename() {
     }
     closedir(dr);
     free(WEBOTS_TMP_DIR);
-    if (number == -1)
+    // No Webots instance has been started
+    if (number == -1) {
+      snprintf(error_buffer, ERROR_BUFFER_SIZE, "Cannot connect to Webots instance");
```

Comment:
We should make this message different from the next one:
```suggestion
      snprintf(error_buffer, ERROR_BUFFER_SIZE, "Cannot find any instance of Webots");
```